src/v2/Apps/Fair/routes.tsx

```diff
@@ -129,17 +129,23 @@ export const routes: RouteConfig[] = [
 ]
 
 function initializeVariablesWithFilterState(params, props) {
-  const initialFilterState = props.location ? props.location.query : {}
+  const initialFilterStateFromUrl = props.location ? props.location.query : {}
+  const camelCasedFilterStateFromUrl = paramsToCamelCase(
+    initialFilterStateFromUrl
+  )
 
   let aggregations: string[] = ["TOTAL", "GALLERY", "MAJOR_PERIOD", "ARTIST"]
   if (props.context.user) aggregations = aggregations.concat("FOLLOWED_ARTISTS")
 
   const state = {
     sort: "-decayed_merch",
-    ...paramsToCamelCase(initialFilterState),
+    ...camelCasedFilterStateFromUrl,
     ...params,
     aggregations,
     shouldFetchCounts: !!props.context.user,
+    includeArtworksByFollowedArtists:
```

Comment:
side-fix, as a logged out person, if you somehow visit a URL with the `?include_artworks_by_followed_artists=true` param, we don't actually want to do that (it will error since you are logged out). Instead we silently ignore it.